ext/webidl/00_webidl.js

```diff
@@ -834,8 +834,7 @@
   }
 
   function createPromiseConverter(converter) {
-    return (V, opts) =>
-      PromisePrototypeThen(PromiseResolve(V), (V) => converter(V, opts));
+    return (V, opts) => PromiseResolve(converter(V, opts));
```

Comment:
This change is needed for:

- https://github.com/web-platform-tests/wpt/blob/0d21c20b40df4f3f1cb6418a2eedd494257543bb/streams/readable-streams/async-iterator.any.js#L439
- https://github.com/web-platform-tests/wpt/blob/0d21c20b40df4f3f1cb6418a2eedd494257543bb/streams/readable-streams/async-iterator.any.js#L460

`createPromiseConverter` is only used in `06_streams.js`, no side effect detected so far.